src/elements/kw-share-card/kw-share-card.html

```diff
@@ -13,131 +12,150 @@
+            }
+            
+            .orangebutton {
+                background-color: #FF8413;
+                border-radius: 5px;
+                width: 79px;
+                height: 32px;
+                color: white;
+                border: none;
+            }
+            
+            .remixbutton {
+                width: 79px;
+            }
+            
+            .bottom .line {
+                height: 1px;
+                width: 100%;
+                display: block;
+                background-color: rgba(99, 107, 126, .24);
             }
-            .stats .likes,
-            .stats .comments {
-                padding: 6px;
-            }
-            div[tombstone].top .avatar {
-                border-radius: 50%;
-                background: rgba(0, 0, 0, 0.01);
-                overflow: hidden;
-                z-index: 0;
-            }
-            div[tombstone].top .title {
-                width: 120px;
-                height: 20px;
-                background: rgba(0, 0, 0, 0.03);
-            }
-            div[tombstone].top .author,
-            div[tombstone].top date-util {
-                width: 120px;
-                height: 20px;
-                background: rgba(0, 0, 0, 0.01);
-                color: transparent;
-            }
+            
             div[tombstone].bottom .likes,
-            div[tombstone].bottom .comments {
+            div[tombstone].bottom .comments,
+            div[tombstone].bottom .shares {
                 width: 80px;
                 height: 20px;
                 background: rgba(0, 0, 0, 0.01);
                 color: transparent;
             }
         </style>
-        <div class="top" tombstone$="[[!share]]">
-            <iron-image class="avatar" src="[[share.user.avatar.urls.circle]]" sizing="contain" preload fade></iron-image>
-            <div class="share-info">
-                <span class="title">[[share.title]]</span>
-                <span class="author">by [[share.user.username]]</span>
-            </div>
-            <div class="datetime">
-                <date-util format="dd Mon yyyy" date="[[_createDate(share.date_created)]]"></date-util>
-            </div>
-        </div>
-        <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade tombstone$="[[!share]]"></iron-image>
-        <div class="bottom" tombstone$="[[!share]]">
-            <div class="stats">
-                <div class="likes">
-                    <span>[[share.likes.length]] likes</span>
-                </div>
-                <div class="comments">
-                    <span>[[share.comments_count]] replies</span>
+        <kw-share-shell id$="[[share.slug]]" share="[[share]]">
+            <div slot="share-card">
+                <iron-image id="cover" class="cover" src="[[share.cover_url]]" sizing="cover" preload fade tombstone$="[[!share]]"></iron-image>
+                <div class="bottom" tombstone$="[[!share]]">
```

Comment:
A `tombstone` attribute on the container and css styling applied, would be more performant as the browser would have to update only one attibute